README.md

```diff
@@ -1,16 +1,33 @@
 # Connectors
-Connectors allows investors to provide liquidity into Centrifuge pools without having to bridge over to the Centrifuge chain. It provides a fully native experience for users on any supported chain.
+Connectors enables seamless deployment of Centrifuge RWA pools on any EVM-compatible blockchain.
```

Comment:
I don't know if I follow this change. The new sentence makes it sound as if Connectors allow's users to deploy pools from any EVM chain we support but really it allows for users to invest. Can you expand on this and why you think the new one is a better pick?